usaspending_api/references/v2/views/autocomplete.py

```diff
@@ -192,3 +194,25 @@ def post(self, request):
         }
 
         return Response(response)
+
+
+class GlossaryAutocompleteViewSet(BaseAutocompleteViewSet):
+
+    @cache_response()
+    def post(self, request):
+
+        search_text, limit = self.get_request_payload(request)
+
+        queryset = Definition.objects.all()
+
+        glossary_terms = queryset.filter(slug__icontains=search_text)
+        serializer = DefinitionSerializer(glossary_terms[:limit], many=True)
+
+        response = {
+            'search_text': search_text,
+            'results': glossary_terms.values_list('term', flat=True)[:limit],
+            'count':  min(glossary_terms.count(), limit),
```

Comment:
Is there a better way to limit the results than by using [:limit] twice and this `min()`?